backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/CAdapterGenerator.kt

```diff
@@ -649,7 +649,7 @@ internal class CAdapterGenerator(val context: Context) : DeclarationDescriptorVi
     override fun visitPackageFragmentDescriptor(descriptor: PackageFragmentDescriptor, ignored: Void?): Boolean {
         val fqName = descriptor.fqName
         val name = if (fqName.isRoot) "root" else translateName(fqName.shortName().asString())
```

Comment:
may be it'd be better to move name lower in lambda for `getOrPut`?

Reply:
Done.